src/main/java/io/meeds/qa/ui/pages/page/factory/tasks/TasksPage.java

```diff
@@ -343,6 +343,9 @@ public class TasksPage extends GenericPage {
   @FindBy(xpath = "//div[@id='projectBoardToolbar']//input")
   private TextBoxElementFacade     searchProjectInput;
 
+  @FindBy(xpath = "//*[@class='v-toolbar__content']//*[@placeholder='Filter by name or description']")
```

Comment:
Wrong XPath:
1. you shouldn't check on placeholder because it can change anytime through crowdin and thus the Test case shouldn't fail when the translation changes
2. avoid using Vuetify specific class `v-toolbar__content`

```suggestion
  @FindBy(xpath = "//*[@class='v-toolbar__content']//*[@placeholder='Filter by name or description']")
```